Stripe/STPRedirectContext.m

```diff
@@ -69,10 +69,10 @@ - (nullable instancetype)initWithPaymentIntent:(STPPaymentIntent *)paymentIntent
         return nil;
     }
 
-    NSString *redirectUrl = nextSourceAction[@"value"][@"url"];
-    return [self initWithNativeRedirectUrl:nil
-                               redirectUrl:[NSURL URLWithString:redirectUrl]
-                                 returnUrl:paymentIntent.returnUrl
+    NSString *redirectURL = nextSourceAction[@"value"][@"url"];
+    return [self initWithNativeRedirectURL:nil
+                               redirectURL:[NSURL URLWithString:redirectURL]
+                                 returnURL:paymentIntent.returnUrl
```

Comment:
After this PR lands, the only case-sensitive search results for `Url` are the `returnUrl` property on `STPPaymentIntent`.

That property is being removed for IOS-1013, so I left it unchanged in this PR.